aura/src/Aura/Languages.hs

```diff
@@ -39,31 +39,19 @@ along with Aura.  If not, see <http://www.gnu.org/licenses/>.
 
 -}
 
-module Aura.Languages where
+module Aura.Languages
+    ( module Aura.Languages
```

Comment:
If this is a way to reexport the two new modules you've added, it would be better to make that explicit a la:

```haskell
( module Aura.Languages.Base
, module Aura.Languages.Fields
...
```